doc/getting_started/how_to_renate_config.rst

```diff
@@ -112,7 +112,20 @@ These are optional as well but, if omitted, Renate will use :code:`train_transfo
 
 .. literalinclude:: ../../examples/getting_started/renate_config.py
     :caption: Example
-    :lines: 75-
+    :lines: 76-
+
+Custom Metrics
+==============
+It is possible to specify a set of custom metrics to be measured during the training process.
+The metrics can be either imported from :code:`torchmetrics`, which offers a vast collection,
+or created ad-hoc by implementing the same interface
+(see this `tutorial https://torchmetrics.readthedocs.io/en/stable/pages/implement.html`_).
+
+To enable the usage of additional metrics in Renate it is sufficient to implement the
+:code:`metrics_fn` function, returning a dictionary where the key is a string containing the
+metric's name and the value is an instantiation of the metric class.
+In the example above we add a metric called :code:`my_accuracy` by instantiating the accuracy
```

Comment:
it's a bit strange to show the metrics_fn in the code snippet above, check here how it looks right now: https://renate.readthedocs.io/en/dev/getting_started/how_to_renate_config.html#transforms

In the transform section we only want to show the transform functions and we should the metrics_fn in the new section.